Sources/XiEditor/Client.swift

```diff
@@ -86,4 +86,7 @@ protocol XiClient: AnyObject {
 
     /// A notification containing the current replace status.
     func replaceStatus(viewIdentifier: String, status: ReplaceStatus)
+    
+    /// A notification telling toggle tail config was successfully changed.
+    func toggleTailConfigChanged(viewIdentifier: String, isTailEnabled: Bool)
```

Comment:
Why is this `toggleTailConfigChanged`? Do you envision the tail config to have more things than whether if the user enables tailing? Even if so, I feel like this should just be something like `enableTailing`.

Reply:
Sure. I'll rename.

Reply:
Ok. I have made this change.